Description: Adds 2 templates for both 32 and 64 bits Wheezy
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no

--- guest-templates-0.1.orig/create_templates.ml
+++ guest-templates-0.1/create_templates.ml
@@ -555,6 +555,7 @@ let debian_template name release archite
 		vM_name_description = bt.vM_name_description ^ (match release with
 			| "lenny"   -> "\nOfficial Debian Lenny CD/DVD images do not support XenServer. To find ISO images that do, please refer to: http://community.citrix.com/display/xs/Debian+Lenny"
 			| "squeeze" -> "\nIn order to install Debian Squeeze from CD/DVD the multi-arch ISO image is required."
+			| "wheezy" -> "\nIn order to install Debian Wheezy from CD/DVD the multi-arch ISO image is required."
 			| _         -> "")
 	}
 
@@ -604,6 +605,8 @@ let create_all_templates rpc session_id
 		debian_template "Debian Lenny 5.0" "lenny" X32 [    ];
 		debian_template "Debian Squeeze 6.0" "squeeze" X32 [    ];
 		debian_template "Debian Squeeze 6.0" "squeeze" X64_debianlike [    ];
+		debian_template "Debian Wheezy 7.0" "wheezy" X32 [    ];
+		debian_template "Debian Wheezy 7.0" "wheezy" X64_debianlike [    ];
 		debian_template "Ubuntu Lucid Lynx 10.04" "lucid" X32 ~supports_cd:false [    ];
 		debian_template "Ubuntu Lucid Lynx 10.04" "lucid" X64_debianlike ~supports_cd:false [    ];
 
