diff --git a/create_templates_main.ml b/create_templates_main.ml
index 122685a..b49602f 100644
--- a/create_templates_main.ml
+++ b/create_templates_main.ml
@@ -17,7 +17,7 @@ open Client
 
 let rpc xml =
     let open Xmlrpc_client in
-    XML_protocol.rpc ~transport:(Unix "/var/xapi/xapi") ~http:(xmlrpc ~version:"1.0" "/") xml
+    XML_protocol.rpc ~transport:(Unix "/var/lib/xcp/xapi") ~http:(xmlrpc ~version:"1.0" "/") xml
 
 let _ =
     let session_id = Client.Session.login_with_password ~rpc ~uname:"" ~pwd:"" ~version:"1.0" in
