diff --git a/61-regenerate-old-templates b/61-regenerate-old-templates
index ec5de12..58f8fc0 100755
--- a/61-regenerate-old-templates.orig
+++ b/61-regenerate-old-templates
@@ -10,7 +10,7 @@ XE=/usr/bin/xe
 # Added to syslog messages from this script
 TAG=templates
 
-. /etc/xensource-inventory
+. /etc/xcp/inventory
 
 UPGRADE="false"
 [ -r ${FIRSTBOOT_DATA_DIR}/host.conf ] && . ${FIRSTBOOT_DATA_DIR}/host.conf
@@ -24,7 +24,7 @@ this_host_is_master(){
     # Therefore HA isn't running and it's safe to peek at the pool.conf. The reason we don't use the
     # the CLI is because, if we're a slave, we want to detect this without blocking and return from
     # this script.
-    pool_role=$(cat < /etc/xensource/pool.conf)
+    pool_role=$(cat < /etc/xcp/pool.conf)
     [ "${pool_role}" = "master" ]
     return $?
 }
